index.bs

```diff
@@ -415,12 +415,19 @@ dictionary UADataValues {
   DOMString uaFullVersion;
 };
 
+dictionary UALowEntropyJSON {
+  FrozenArray&lt;NavigatorUABrandVersion&gt; brands;
```

Comment:
The use of `FrozenArray` in dictionaries is the topic of https://github.com/heycam/webidl/issues/900. If there isn't really a reason to use it here, then `sequence<NavigatorUABrandVersion> brands` would be more straightforward here.